### ISSUE TITLE
False positive: Y015

### ISSUE THREAD

**Issue by pmmmwh** [charliermarsh/ruff#3862]
## Reproduction

```pyi
from typing import TypeVar

from typing_extensions import ParamSpec, TypeVarTuple

_T = TypeVar("_T")
_P = ParamSpec("_P")
_TV = TypeVarTuple("_TV")

```

### Current Behaviour

On autofix, `TypeVar` will be replaced by ellipsis:

```pyi
_T = ...
_P = ...
_TV = ...

```

### Expected

Keep the original code untouched and should not report them as issues.

Perhaps the implementation should also whitelist `TypeVar`, `TypeVarTuple` and `ParamSpec`?

https://github.com/charliermarsh/ruff/blob/924bebbb4a0e1a9002a0274cbb58fc85335089dc/crates/ruff/src/rules/flake8_pyi/rules/simple_defaults.rs#L340-L342

## Settings

```toml
[tool.ruff]
fix = true
select = ["PYI"]
```

## Version

```
ruff 0.0.260
```

**Comment by pmmmwh** [comment]
Duplicate of #3860
